com.markopejic.downloader.json

```diff
@@ -0,0 +1,120 @@
+{
+    "id": "com.markopejic.downloader",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "command": "com.markopejic.downloader",
+    "finish-args": [
+        "--share=network",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--device=dri",
+        "--filesystem=xdg-download:create"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "cpython",
```

Comment:
Python is in the runtime.

Reply:
Removed the module in [69f4640](https://github.com/markop404/flathub/commit/69f46402002da193d6f75e74de82f5ad06cfb043)